components/ipfs/ipfs_utils.cc

```diff
@@ -273,4 +273,19 @@ bool TranslateIPFSURI(const GURL& url,
   return false;
 }
 
+GURL ResolveWebUIFilesLocation(const std::string& directory,
+                               version_info::Channel channel) {
+  GURL url = GetAPIServer(channel);
+  GURL::Replacements replacements;
+  replacements.SetPathStr("/webui/");
+  std::string webui_files_ref = std::string("/files") + directory;
+  replacements.SetRefStr(webui_files_ref);
+  return url.ReplaceComponents(replacements);
+}
+
+bool IsIpfsMenuEnabled(content::BrowserContext* browser_context) {
+  return ipfs::IsIpfsEnabled(browser_context) &&
```

Comment:
nit: You can just use IsIpfsEnabled(browser_context) && IsLocalGatewayConfigured(browser_context) here.